lightly/api/__init__.py

```diff
@@ -2,12 +2,19 @@
 
 # Copyright (c) 2020. Lightly AG and its affiliates.
 # All Rights Reserved
-
+from lightly.api import patch as _patch
 from lightly.api.api_workflow_client import ApiWorkflowClient
 from lightly.api.api_workflow_artifacts import ArtifactNotExist
-from lightly.api.patch_rest_client import patch_rest_client
 
-from lightly.openapi_generated.swagger_client.rest import RESTClientObject
+from lightly.openapi_generated.swagger_client import ApiClient as _ApiClient, Configuration as _Configuration
+from lightly.openapi_generated.swagger_client.rest import RESTClientObject as _RESTClientObject
```

Comment:
Made these imports private so they cannot be imported with `from lightly.api import Configuration` or similar.